src/preview.js

```diff
@@ -30,17 +30,20 @@ const getReadOnWikiCta = ( lang, title, isTouch ) => {
 	return `<a href="${ buildWikipediaUrl( lang, title, isTouch ) }" target="_blank" class="wikipediapreview-cta-readonwiki">${ msg( lang, 'read-on-wiki' ) }</a>`
 }
 
-const render = ( lang, isTouch, dir, headerContent, bodyContent, prefersColorScheme ) => {
+const render = (
+	lang, isTouch, dir, headerContent, bodyContent, prefersColorScheme, withArrow = true
```

Comment:
Is there anyway we could make the arrow part of the popup component instead of the preview?

Reply:
Done, please check it again.